pwd
mkd OlaitanTayo
mkdir OlaitanTayo
cd OlaitanTayo/
mkdir biocomputing
rm biocomputing/
rmdir biocomputing/
ls
mkdir biocomputing | cd biocomputing
mkdir biocomputing; cd biocomputing
ls
cd O
cd . .
cd ./
cd .
cd /
ls
cd ~/OlaitanTayo/
ls
rmdir biocomputing/
ls
mkdir biocomputing; cd biocomputing
wget https://raw.githubusercontent.com/josoga2/dataset-repos/main/wildtype.fna 
ls
wget https://raw.githubusercontent.com/josoga2/dataset-repos/main/wildtype.gbk
curl -o https://raw.githubusercontent.com/josoga2/dataset-repos/main/wildtype.gbk
curl -0 https://raw.githubusercontent.com/josoga2/dataset-repos/main/wildtype.gbk
ls
wget https://raw.githubusercontent.com/josoga2/dataset-repos/main/wildtype.gbk
ls
mv wildtype.fna OlaitanTayo
ls
man mv
ls
mv -t OlaitanTayo OlaitanTayo 
rm OlaitanTayo 
ls
wget https://raw.githubusercontent.com/josoga2/dataset-repos/main/wildtype.fna
ls
mv/~/OlaitanTayo/biocomputing/wildtype.fna /~OlaitanTayo/
mv/home/OlaitanTayo/biocomputing/wildtype.fna /home/OlaitanTayo/
mv /home/OlaitanTayo/biocomputing/wildtype.fna /home/OlaitanTayo/
mv /~/OlaitanTayo/biocomputing/wildtype.fna /~/OlaitanTayo/
mv wildtype.fna /~/OlaitanTayo/
mv wildtype.fna ~/OlaitanTayo/
ls
cd OlaitanTayo
cd..
cd .
cd ./
cd ..
ls
cd biocomputing/
rm wildtype.gbk
ls
grep tata wildtype.gbk.1 
grep tatatata wildtype.gbk.1 
grep tatatata wildtype.gbk.1 > mutant
ls
head mutant 
wget https://www.ncbi.nlm.nih.gov/nuccore/NC_001803.1?report=fasta&from=5619&to=7521
ls
head wget_log
wget https://www.ncbi.nlm.nih.gov/nuccore/NC_001803.1?report=fasta&from=5619&to=7521# > RSV
ls
head RSV
tail RSV
ls
wget https://www.ncbi.nlm.nih.gov/nuccore/NC_001803.1?report=fasta > RSV1
ls
head NC_001803.1\?report\=fasta
head RSV1
cat RSV1
nano RSV1
wget https://www.ncbi.nlm.nih.gov/nuccore/NC_001803.1?report=fasta
grep caat NC_001803.1\?report\=fasta
grep caat NC_001803.1\?report\=fasta.2
grep caat web_log
rm NC_001803.1\?report\=fasta
rm NC_001803.1\?report\=fasta.
rm NC_001803.1\?report\=fasta.1
rm NC_001803.1\?report\=fasta.2
ls
rm wget-log.1
rm RSV
rm RSV1 
rm wget-log 
ls
curl -0 https://www.ncbi.nlm.nih.gov/nuccore/NC_001803.1?report=fasta&from=5619&to=7521#
wget "https://www.ncbi.nlm.nih.gov/nuccore/NC_001803.1?report=fasta&from=5619&to=7521" -O RSV_F_gene.fasta
ls
head RSV_F_gene.fasta 
cat RSV_F_gene.fasta 
rm wget-log.1
wget "https://www.ncbi.nlm.nih.gov/gene?linkname=gene_gene_h3k4me3&from_uid=1494475#" -O RSV_F_gene
head RSV_F_gene
ls
rm R
rm RSV_F_gene
rm RSV_F_gene.fasta 
wget "https://www.ncbi.nlm.nih.gov/gene?linkname=gene_gene_h3k4me3&from_uid=1494475#" -O RSV_F_gene.fasta
less RSV_F_gene.fasta 
rm RSV_F_gene.fasta 
wget "https://www.ncbi.nlm.nih.gov/nuccore/NC_001803.1?from=5619&to=7521&report=fasta" -O RSV_F_gene.fasta
head RSV_F_gene.fasta 
efetch -db nucleotide -id NC_001803.1 -seq_start 5619 -seq_stop 7521 -format fasta > RSV_F_gene.fasta1
sudo apt-get install -y entrez-direct
efetch -db nucleotide -id NC_001803.1 -seq_start 5619 -seq_stop 7521 -format fasta > RSV_F_gene.fasta1
sudo apt-get install -y entrez-direct
cd ~/downloads
cd ~
ls
cd ~/Downloads
wget "https://www.ncbi.nlm.nih.gov/gene/1494467"
ls
head 1494467 
wget "https://www.ncbi.nlm.nih.gov/nuccore/NC_001803.1?report=fasta"
ls
head 'NC_001803.1?report=fasta.1'
cd O
cd OlaitanTayo/
cd biocomputing/
ls
rm RSV_F_gene.fasta
rm RSV_F_gene.fasta1 
ls
head sequence.fasta 
tail -n +2 sequence.fasta | wc -l
tail -n +2 sequence.fasta | grep -o "A" | wc -l
tail -n +2 sequence.fasta | grep -o "G" | wc -l
tail -n +2 sequence.fasta | grep -o "C" | wc -l
tail -n +2 sequence.fasta | grep -o "T" | wc -l
total_bases=$(tail -n +2 sequence.fasta | grep -o "[ACGT]" | wc -l)
gc_bases=$(tail -n +2 sequence.fasta | grep -o "[GC]" | wc -l)
gc_content=$(echo "scale=2; $gc_bases / $total_bases * 100" | bc)
echo "GC content: $gc_content%"
sudo apt-get update && sudo apt-get install bc
which bc
total_bases=$(tail -n +2 sequence.fasta | grep -o "[ACGT]" | wc -l)
gc_bases=$(tail -n +2 sequence.fasta | grep -o "[GC]" | wc -l)
gc_content=$(echo "scale=2; $gc_bases / $total_bases * 100" | bc)
echo "GC content: $gc_content%"
nano OlaitanTayo.fasta
echo "Number of A:-n +2 sequence.fasta | grep -o "A" 
| wc -l

echo "GC content: $gc_content%"



"
pwd
ls
cd OlaitanTayo/
ls
cd biocomputing/
echo "A: $(-n +2 sequence.fasta | grep -o "A" | wc -l)" >> OlaitanTayo.fasta
echo "G: $(-n +2 sequence.fasta | grep -o "G" | wc -l)" >> OlaitanTayo.fasta
echo "T: $(-n +2 sequence.fasta | grep -o "T" | wc -l)" >> OlaitanTayo.fasta
echo "C: $(-n +2 sequence.fasta | grep -o "C" | wc -l)" >> OlaitanTayo.fasta
LS
ls
-n 
-n +2 sequence.fasta | grep -o "G"
echo "A: $(tail -n +2 sequence.fasta | grep -o "A" | wc -l)" >> OlaitanTayo.fasta
echo "T: $(tail -n +2 sequence.fasta | grep -o "T" | wc -l)" >> OlaitanTayo.fasta
echo "G: $(tail -n +2 sequence.fasta | grep -o "G" | wc -l)" >> OlaitanTayo.fasta
echo "C: $(tail -n +2 sequence.fasta | grep -o "C" | wc -l)" >> OlaitanTayo.fasta
cd OlaitanTayo
cd OlaitanTayo.fasta
nano OlaitanTayo.fasta
git --version
git clone https://github.co/Olaitan2413/your-repo.git
git clone https://github.co/Olaitan2413/HackBio-stage-3.git
cd HackBio-stage-3
git clone https://github.com/Olaitan2413/your-repo.git
ssh-keygen -t ed25519 -C "ayegbusiolaitan@yahoo.com"
cat ~/.ssh/id_ed25519.pub
git clone git@github.com:Olaitan2413/HackBio-stage-3.git
eval "$(ssh-agent -s)"
ssh-add ~/.ssh/id_ed25519
cat ~/.ssh/id_ed25519.pub
ssh -T git@github.com
rm ~/.ssh/id_ed25519 ~/.ssh/id_ed25519.pub
ssh-keygen -t ed25519 -C "ayegbusiolaitan@yahoo.com"
eval "$(ssh-agent -s)"
ssh-add ~/.ssh/id_ed25519
cat ~/.ssh/id_ed25519.pub
ssh -T git@github.com
git clone git@github.com:Olaitan2413/HackBio-stage-3.git
cd HackBio-stage-3/
mkdir output
ls
mv /OlaitanTayo/biocomputing/OlaitanTayo.fasta output/
cd ..
ls
mv /OlaitanTayo/biocomputing/OlaitanTayo.fasta output
nano sequence.fasta 
nano OlaitanTayo.fasta
ls
echo "A: $(tail -n +2 OlaitanTayo.txt | gr
ep -o "A" | wc -l)" >> OlaitanTayo.txt                 
echo "A: $(tail -n +2 OlaitanTayo.txt | grep -o "A" | wc -l)" >> OlaitanTayo.txt                 
nano OlaitanTayo.txt
echo "A: $(tail -n +2 OlaitanTayo.txt | grep -o "A" | wc -l)" >> OlaitanTayo.txt                 
echo "G: $(tail -n +2 OlaitanTayo.txt | grep -o "G" | wc -l)" >> OlaitanTayo.txt                 
echo "T: $(tail -n +2 OlaitanTayo.txt | grep -o "T" | wc -l)" >> OlaitanTayo.txt                 
echo "C: $(tail -n +2 OlaitanTayo.txt | grep -o "C" | wc -l)" >> OlaitanTayo.txt                 
nano OlaitanTayo.txt
ls
rm OlaitanTayo.fasta
rm OlaitanTayo.fastaq 
cd HackBio-stage-3/
mv ~/OlaitanTayo/biocomputing/OlaitanTayo.txt output/
ls
cd output
ls
git status
git add
"git add output"
git log --oneline
git push origin output
git push origin HackBio-stage-3
git push origin main
git branch
git log --oneline
git push origin main
git branch
pwd
cd OlaitanTayo/biocomputing/
ls
cd HackBio-stage-3/
ls
ls output
git add output/OlaitanTayo.txt
git commit -m "Added file to /output folder"
git config --global user.name "OlaitanTayo"
git config --global user.email "ayegbusiolaitan@yahoo.com"
git config --global user.name "OlaitanTayo2413"
git config --global user.email "ayegbusiolaitan@yahoo.com"
git config user.name "OlaitanTayo2413"
git config user.email "ayegbusiolaitan@yahoo.com"
git commit -m "Added file to /output folder"
git push origin main
